csrankings.csv

```diff
@@ -14478,6 +14483,7 @@ Per Runeson,Lund University,http://cs.lth.se/per_runeson,7B58FysAAAAJ
 Per Stenström,Chalmers/GU,https://www.chalmers.se/en/staff/Pages/per-stenstrom.aspx,wsMoM6cAAAAJ
 Per Wahlund,Uppsala University,https://katalog.uu.se/empinfo/?id=XX2255,NOSCHOLARPAGE
 Peraphon Sophatsathit,Chulalongkorn University,https://cop.car.chula.ac.th/%E0%B8%9E%E0%B8%B5%E0%B8%A3%E0%B8%B0%E0%B8%9E%E0%B8%B1%E0%B8%99%E0%B8%98%E0%B9%8C,NOSCHOLARPAGE
+Percy Liang,Stanford University,https://cs.stanford.edu/~pliang/,pouyVyUAAAAJ

```

Comment:
At Microsoft.

Reply:
https://blogs.microsoft.com/blog/2018/05/20/microsoft-acquires-semantic-machines-advancing-the-state-of-conversational-ai/

See `old/industry.csv`. If he has returned to a full-time (>= 75%) position at Stanford, he can be reinstated.